src/AVKit/Enums.cs

```diff
@@ -31,16 +31,32 @@ public enum AVPlayerViewTrimResult : long {
 		OKButton,
 		CancelButton
 	}
+	
+	[Mac (10,15)]
+	[Native]
+	public enum AVRoutePickerViewButtonState : long
+	{
+		Normal,
+		NormalHighlighted,
+		Active,
+		ActiveHighlighted,
+	}
 #endif
 
-#if !TVOS && (!MONOMAC || !XAMCORE_4_0)
-	[iOS (9,0)]
+#if !MONOMAC || !XAMCORE_4_0
+	[iOS (9,0), TV (13,0)]
 	[Native]
 	[ErrorDomain ("AVKitErrorDomain")]
 	public enum AVKitError : long {
 		None = 0,
 		Unknown = -1000,
-		PictureInPictureStartFailed = -1001
+		PictureInPictureStartFailed = -1001,
+		[TV (13,0)]
+		ContentRatingUnknown = -1100,
+		[TV (13,0)]
+		ContentDisallowedByPasscode = -1101,
+		[TV (13,0)]
+		ContentDisallowedByProfile = -1102,
```

Comment:
all those new members will also be new in iOS 13 but...
we do not add availability attributes on error codes (not useful, they are output only and you can't ignore them if your code will be run on newer OS).